src/pages/[platform]/build-a-backend/storage/upload/index.mdx

```diff
@@ -42,6 +42,17 @@ export function getStaticProps(context) {
   };
 }
 
+<InlineFilter filters={["android", "swift", "flutter"]}>
+
+<Callout info>
+The latest version of Amplify Storage supports specifying S3 objects as a paths.  
+We recommend using `path` instead of `key` to specify S3 objects.
+
+Note: `key` parameter is deprecated and maybe removed in next major version.
```

Comment:
Same np as above: Pretty sure this should be `may be` and not `maybe`.

Reply:
Should stop copying and pasting the same mistake over and over, huh? Thanks!